SonarQube.Bootstrapper/Resources.resx

```diff
@@ -129,23 +129,29 @@
   <data name="ERROR_CmdLine_UrlRequired" xml:space="preserve">
     <value>The SonarQube URL must be specified in a settings file or on the command line (e.g. using /d:sonar.host.url=http://myserver:9000 )</value>
   </data>
-  <data name="ERROR_CouldNotFindIntegrationZip" xml:space="preserve">
```

Comment:
The start of this message is too specific: there are other possible failures e.g. bad url, server is down, authentication errors. I've made the first part a bit more general.